toolkit/tools/srpmpacker/srpmpacker.go

```diff
@@ -602,11 +828,12 @@ func hydrateFiles(fileTypeToHydrate fileType, specFile, workingDir string, srcCo
 
 	for fileNeeded, alreadyHydrated := range fileHydrationState {
 		if !alreadyHydrated {
-			logger.Log.Panicf("Unable to hydrate file: %s", fileNeeded)
+			err = fmt.Errorf("unable to hydrate file: %s", fileNeeded)
+			logger.Log.Error(err)
```

Comment:
We used to panic, now we continue. Is thsi by design? Or should we at least break?

Reply:
See above comment. It is up to the routine orchestrating the workers to determine what merits a break.